libraries/Microsoft.Bot.Builder.Dialogs.Adaptive.Teams/Schemas/Actions/Teams.SendTabCardResponse.schema

```diff
@@ -0,0 +1,36 @@
+{
+    "$schema": "https://schemas.botframework.com/schemas/component/v1.0/component.schema",
+    "$role": "implements(Microsoft.IDialog)",
+    "title": "Tab card response",
+    "description": "Send n tab response containing cards.",
```

Comment:
>n [](start = 25, length = 1)

a

Reply:
Shouldn't this be 'a'?

---
In reply to: [574900776](https://github.com/microsoft/botbuilder-dotnet/pull/5185#discussion_r574900776) [](ancestors = 574900776)